refactor-example.py

```
@@ -0,0 +1,28 @@
+def messy_read_and_process(file_path, callback):
+    import string
+
+    try:
+        with open(file_path, 'r', encoding='utf-8') as f:
+            data = f.read()
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)